README.md

```diff
@@ -382,19 +380,19 @@ make build_docker
 Run the following command to enter the interactive CLI:
 
 ```sh
-docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/staking-deposit-cli
+docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/ethstaker-deposit-cli
```

Comment:
I think the image name needs to be `eth-educators/ethstaker-deposit-cli` (from the new `DOCKER_IMAGE` in Makefile). This will need more tests from #61 .

Reply:
Good catch

Reply:
ack